core/src/modules/caches/propertiesCache.ts

```diff
@@ -0,0 +1,68 @@
+import { ModelCache } from "../modelCache";
+import { Property } from "../../models/Property";
+import { Source } from "../../models/Source";
+import { Option } from "../../models/Option";
+import { Filter } from "../../models/Filter";
+import { Includeable, WhereAttributeHash } from "sequelize";
+
+function getInclude() {
+  const include: Includeable[] = [
+    { model: Source.unscoped(), required: false },
+    { model: Option, required: false },
+    { model: Filter, required: false },
+  ];
+  return include;
```

Comment:
Seems like this could just be a const? Unless `Source.unscoped()` has some call-time context?

Reply:
I added a note - it needs to be looked-up at runtime to prevent circular deps.